src/core/models/monitor.py

```diff
@@ -20,6 +20,7 @@ class MsiMonitor(DataClassDictMixin):
     timestamp_field: str = field(default=None, metadata={"sa": Column(String(100))})
     workspace: str = field(default=None, metadata={"sa": Column(String(100))})
     source: str = field(default=None, metadata={"sa": Column(String(100))})
+    days_ago: int = field(default=100, metadata={"sa": Column(Integer)})
```

Comment:
does this automatically update the expectations of the API request body?

Reply:
Yes, thanks to the magic of DataClassDictMixin and SQLAlchemy